### PR TITLE
Define how "Assert:" works

### PR DESCRIPTION
**Pull request by annevk**
See #87 for direct inspiration, though we’ve been using this here and
there for a while now.

***
[Preview](https://infra.spec.whatwg.org/branch-snapshots/annevk/assertions/) | [Diff](https://s3.amazonaws.com/pr-preview/whatwg/infra/139484a...3b61f95.html)

### CODE REVIEW

File: infra.bs [hunk 1]
```diff
@@ -203,6 +203,18 @@ iteration.
 </div>
 
 
+<h3 id=assertions>Assertions</h3>
+
+<p>To improve readability, it can sometimes help to add assertions to algorithms, stating
+invariants. To do this, write "<code>Assert: </code>", followed by a statement that must be true.
```
Comment (domenic) [inline infra.bs]: This should not be `<code>`; just normal font is fine.

**Comment by annevk** [comment]
Shall I add that as a note?

File: infra.bs (hunk 1 above)
Comment (annevk) [inline infra.bs]: Elsewhere we say that strings should use code.

Comment (annevk) [inline infra.bs]: But I guess we haven't done it for algorithm keywords per se.

Comment (domenic) [inline infra.bs]: Yeah this isn't a string in the sense of the data type.

**Comment by domenic** [comment]
Not sure if it's a note or normative; I could go either way.

**Comment by annevk** [comment]
Added a bit.

**Pull request merged**